src/GameQ/Protocols/Rust.php

```diff
@@ -43,22 +43,11 @@ class Rust extends Source
      */
     protected $name_long = "Rust";
     
-    /**
-     * Overload so we can get max players from mp of keywords and num players from cp keyword
+	/**
+     * query_port = client_port + 1
      *
-     * @param Buffer $buffer
+     * @type int
      */
-    protected function processDetails(Buffer $buffer)
```

Comment:
What's the reason for removing this function?

Reply:
With the function I have no online status